chromium_src/chrome/browser/ui/views/bookmarks/bookmark_menu_delegate.cc

```diff
@@ -3,8 +3,27 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
+#include "chrome/browser/ui/views/bookmarks/bookmark_menu_delegate.h"
+
 #include "brave/browser/ui/views/bookmarks/brave_bookmark_context_menu.h"
 
 #define BookmarkContextMenu BraveBookmarkContextMenu
+#define GetMaxWidthForMenu GetMaxWidthForMenu_ChromiumImpl
 #include "src/chrome/browser/ui/views/bookmarks/bookmark_menu_delegate.cc"
+#undef GetMaxWidthForMenu
 #undef BookmarkContextMenu
+
+namespace {
+
+const int kBraveMaxMenuWidth = 800;
```

Comment:
```suggestion
constexpr int kBraveMaxMenuWidth = 800;
```

Reply:
Done in [e844d26](https://github.com/brave/brave-core/commit/e844d269d4d23978a06b62dd155fa57d3bf4dd69)